lib/travis/build/appliances/disable_ssh_roaming.rb

```diff
@@ -5,7 +5,7 @@ module Build
     module Appliances
       class DisableSshRoaming < Base
         def apply
-          sh.if "$(sw_vers -productVersion | cut -d . -f 2) -lt 12" do
+          sh.if %("$(sw_vers -productVersion 2>/dev/null | cut -d . -f 2)" -lt 12) do
```

Comment:
In my tests, I found that `[[ "" -lt 12 ]]` evaluates to true, but `[[ -lt 12 ]]` is an error, which is why the subshell is wrapped in `"`.